### PR TITLE
Applied Marc's feedback

### LINKED ISSUE #106: add to "Getting started md"
@shreythecray 
> If I try to go to the next step without adding the secret (e.g. if I forget) then I can't actually tell why I can't move forward:

It'll tell you the problem if you click on the "build." For explaining how to fix the errors, should we add an MD file in the repo?

_Originally posted by @emsesc in https://github.com/bitprj/Intro-To-Serverless/issues/69#issuecomment-830637452_

### LINKED ISSUE #169: Week 1 Feedback from Marc
## Week 1 Step 1

Ok

## Week 1 Step 2

I had my function return "hello world" (lowercase) intentionally to check what it would do.

The GitHub action fails but the student will only watch the issue (I think). So there doesn't seem to be any feedback when the student submits an incorrect answer.

## Week 1 Step 3

Incorrect link to Cataas.com.

# Week 1 Step 4

Ok

# Week 1 Step 5

I didn't end my message with a dot (.) so the test failed. Perhaps do a test using a contains instead of equals and also ignore casing?

Also the error message in [the failed Action](https://github.com/marcduiker/intro-to-serverless-bitprj/runs/2701532573?check_suite_focus=true) was not very clear:

```text
Try again!
We go letmein and incorrect, but it should be "Access granted." and "Access denied."
```

I updated my local code but didn't deploy it yet. So the build fails again.
I deployed the function again but this (obviously) doesn't trigger a new build. So in order for this step to pass I need to do an unnecessary commit/push. It would be nice if a manual trigger was added to the GitHub Actions.

# Week 1 Step 6

### PR DESCRIPTION
**Pull request by emsesc**
- add more documentation about actions + moving on
- edited test for clarity

Closes #169 
Closes #106

### CODE REVIEW

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -64,4 +64,42 @@ Click on `Issues` to find your first Week in the camp: `Getting Started`.
...
+Now, go ahead and click on ":x: build"
+![image](https://user-images.githubusercontent.com/69332964/120933055-68435700-c6c6-11eb-88a9-af248114c6a9.png)
+
+The step that failed **should expand** and you'll see feedback at the bottom. 
+![image](https://user-images.githubusercontent.com/69332964/120933130-a9d40200-c6c6-11eb-91a6-c6a477aa62dc.png)
+
+For this one, the feedback was:
+```
+Try again!
+We go letmein and incorrect, but it should be "Access granted." and "Access denied."
```
Comment (marcduiker) [inline GETTING_STARTED.md:87]: I suggest to use the improved message here: 
`We submitted "letmein" and got ${correct}, which should equal "Access granted."`

**Review by marcduiker — CHANGES_REQUESTED** [review]
One small change to the GETTING_STARTED.md to use the better error message.

File: GETTING_STARTED.md (hunk 1 above)
Comment (emsesc) [reply GETTING_STARTED.md:87]: @marcduiker Sounds good! I did change the error message on the test, but not on the instructions. Thanks 👍

**Comment by emsesc** [comment]
@marcduiker I replaced the feedback example with [this failed workflow](https://github.com/marcduiker/intro-to-serverless-bitprj/runs/2701415270?check_suite_focus=true) since I currently don't have a screenshot with the updated password checking feedback.

**Review by lazyplatypus — APPROVED** [review]

**Pull request merged**